MergerLogic/ImageProcessing/TileMerger.cs

```diff
@@ -18,9 +18,21 @@ public TileMerger(ITileScaler tileScaler, ILogger<TileMerger> logger)
             this._tileScaler = tileScaler;
         }
 
-        public Tile? MergeTiles(List<CorrespondingTileBuilder> tiles, Coord targetCoords, TileFormatStrategy strategy)
+        public Tile? MergeTiles(List<CorrespondingTileBuilder> tiles, Coord targetCoords, TileFormatStrategy strategy, bool uploadOnly = false)
         {
-            var images = this.GetImageList(tiles, targetCoords);
+            if(uploadOnly) {
+                // Ignore target if in upload only mode
+                tiles = tiles.Skip(1).ToList();
+
+                // In case there is only one source then we can just return the data as is
+                if(tiles.Count == 1) {
+                    Tile? rawTile = tiles[0]();
```

Comment:
if only one source to insert and it is a new source - should "shouldUpscale" be set to false as it is now?  or in this conditions we set should upscale to true?
ingestion - improve serving preformance but will increase size
export - will create tiles that don't exists in original data (probably a bug...)